### ISSUE TITLE
Account settings menu is not keyboard accessible

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3499]
**Issue Description**: Cannot use the items in the account settings menu from the account menu button with the keyboard. When I open them menu and hit tab the focus moves to "Log Out" briefly and then the menu closes. 

**WCAG Issue**: [SC 2.1.1 Keyboard](https://www.w3.org/WAI/WCAG20/Understanding/keyboard)

**Notes**: It appears that the menu collapses whenever focus is not on the menu button. The expected UI is something like this pattern from [ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions-active-descendant/)

**Screen recording**: https://share.zight.com/d5uXZ62K

<img width="921" alt="Screenshot 2024-07-31 at 3 10 59 PM" src="https://github.com/user-attachments/assets/356df825-86bb-4f2e-82ce-aaaf360a3baa">
